assets/js/components/notifications/UnsatisfiedScopesAlert.js

```diff
@@ -58,12 +59,13 @@ function mapScopesToModuleNames( scopes ) {
 }
 
 export default function UnsatisfiedScopesAlert() {
+	const isNavigating = useSelect( ( select ) => select( CORE_LOCATION ).isNavigatingTo( /\/o\/oauth2/i ) );
```

Comment:
This is not fully accurate and doesn't fix the underlying issue, since Site Kit redirects to a WP Admin URL containing a `googlesitekit_connect=1` query parameter. Only from there the user is redirected to the authentication service's OAuth proxy endpoint.

It's a good idea to keep the `/o/oauth2` in here to be compatible, should we switch to using OAuth in a more typical way, but we need to also support the real URL Site Kit uses for this purpose.